aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>I think this cell needs to have the correct type set and be rerun to display the clipped Zernikes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Fixed.</p>